gcsio/src/test/java/com/google/cloud/hadoop/gcsio/MockGoogleCloudStorageImplFactory.java

```diff
@@ -0,0 +1,34 @@
+package com.google.cloud.hadoop.gcsio;
+
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.storage.Storage;
+import com.google.cloud.hadoop.util.RetryHttpInitializer;
+import com.google.cloud.hadoop.util.RetryHttpInitializerOptions;
+import com.google.cloud.hadoop.util.testing.FakeCredentials;
+
+class MockGoogleCloudStorageImplFactory {
```

Comment:
Probably better to add this class to the `testing` package: https://github.com/GoogleCloudDataproc/hadoop-connectors/tree/master/gcsio/src/main/java/com/google/cloud/hadoop/gcsio/testing

Reply:
It uses `com.google.cloud.hadoop.util.testing.FakeCredentials`, which is under `testing` source root. We need to move both of them if we want to move it into `main` source root.

Reply:
Or maybe lets move it into the same testing package but under `testing` source root.